zerver/lib/exceptions.py

```diff
@@ -656,3 +657,15 @@ def __init__(self) -> None:
     @override
     def msg_format() -> str:
         return _("You do not have permission to use topic wildcard mentions in this topic.")
+
+
+class PreviousSettingValueMismatchedError(JsonableError):
+    code: ErrorCode = ErrorCode.MISMATCHED_PREVIOUS
+
+    def __init__(self) -> None:
+        pass
+
+    @staticmethod
+    @override
+    def msg_format() -> str:
+        return _("Someone else modified this setting at the same time.")
```

Comment:
Not sure what should be the message and the class name here. Used one of the error code suggested in CZO for now, but we can change after more discussion.

Reply:
Yeah post in "api design" asking for suggests text, I think we can do better but seems not a blocker.